test/certify/test_certify.py

```diff
@@ -634,10 +635,17 @@ def test_certify_jwst_invalid_asdf(jwst_serverless_state, jwst_data, caplog):
         CertifyScript(argv)()
         out = caplog.text
 
-    expected_out = f"""Certifying '{jwst_data}/invalid.asdf' (1/1) as 'ASDF' relative to context 'jwst.pmap'
+    if asdf.__version__ < "3.0.0":
+        expected_out = f"""Certifying '{jwst_data}/invalid.asdf' (1/1) as 'ASDF' relative to context 'jwst.pmap'
 instrument='UNKNOWN' type='UNKNOWN' data='{jwst_data}/invalid.asdf' ::  Validation error : Input object does not appear to be an ASDF file or a FITS with ASDF extension
 ########################################
 1 errors
+0 warnings"""
+    else:
+        expected_out = f"""Certifying '{jwst_data}/invalid.asdf' (1/1) as 'ASDF' relative to context 'jwst.pmap'
+instrument='UNKNOWN' type='UNKNOWN' data='{jwst_data}/invalid.asdf' ::  Validation error : Does not appear to be a ASDF file.
```

Comment:
This warning message was changed as asdf 3.0 no longer support FITS files: https://asdf.readthedocs.io/en/3.0.0/asdf/whats_new.html#removed-features